doc/user/content/cloud/create-deployments.md

```diff
@@ -9,11 +9,15 @@ menu:
 
 {{< cloud-notice >}}
 
-On the [Deployments](https://cloud.materialize.com/deployments) page, click **Create deployment**.
+This functionality is only available to [administrators](administer-workspace).
 
-Materialize creates the deployment and assigns it a name and hostname. This should only take about a minute and a half.
+1. On the [Deployments](https://cloud.materialize.com/deployments) page, click **+New deployment**.
 
-The free version of Materialize Cloud allows two deployments. If you need more, please [contact us](../support).
+1. Select the desired deployment size and click **Create**.
```

Comment:
@msudberg 

Default deployment size on creation flow for Open Beta is XS, and they can upgrade (vertically scale) deployments up to S for free. If users know in advance that for their use case they need bigger than size S deployments, they can/ should reach out to our team to discuss details and place the 'order'.